winlogbeat/etc/fields.yml

```diff
@@ -228,7 +228,7 @@ eventlog:
       description: The version number of the event's definition.
 
     - name: xml
-      type: string
```

Comment:
I think keyword is good here. I don't expect the XML will need to be analyzed. I do wonder about the ignore_above value. That might be a bit to small for some of the XML messages.